src/DynamoCore/Models/DynamoModelEvents.cs

```diff
@@ -371,6 +371,11 @@ public virtual void OnEvaluationCompleted(object sender, EvaluationCompletedEven
                 Action showFailureMessage = () => DisplayEngineFailureMessage(e.Error);
                 OnRequestDispatcherBeginInvoke(showFailureMessage);
             }
+            else
+            {
+                // TODO: DYN-6050 This is not the final place to check this property, this is only for Testing                
+                this.Logger.Log("The Workspace is valid for FDX : " + ((HomeWorkspaceModel)CurrentWorkspace).IsValidForFDX);
+            }
```

Comment:
hi @jesusalvino I dont think we need to consider implementing multiple checks because we are supposed to send the graph after the check